File: Source/CardGame.Abstractions/Bus/CardRevealed.cs
using System;

namespace CardGame.CommonModel.Bus
{
    public class CardRevealed : IEvent
    {
        public Guid CorrelationId { get; set; }
        public Guid GameId { get; set; }
        public Guid PlayerId { get; set; }
        public int TargetCardStrength { get; set; }
        public int TargetCardVariant { get; set; }
        public Guid TargetId { get; set; }
    }
}

File: Source/cardgame-client/src/app/commonState/CardRevealed.ts
import { CardStrength } from '../domain/card/CardStrength';


export class CardRevealed {
    readonly correlationId: string;
    readonly gameId: string;
    readonly playerId: string;
    readonly targetCardStrength: CardStrength;
    readonly targetCardVariant: number;
    readonly targetId: string;
}


